src/lib/Select.svelte

```diff
@@ -1,90 +1,316 @@
+    const [floatingRef, floatingContent, floatingUpdate] = createFloatingActions(_floatingConfig);
+
+    onMount(() => {
+        if (listOpen) focused = true;
+        if (focused && input) input.focus();
+    });
+    $effect.pre(() => {
+        placeholderText =
+            placeholderAlwaysShow && multiple
+                ? placeholder
+                : multiple && value?.length === 0
+                  ? placeholder
+                  : value
+                    ? ''
+                    : placeholder;
+    });
+    $effect.pre(() => {
+        filterText;
+        value;
+        items;
+        untrack(
+            () =>
+                (filteredItems = filter({
+                    loadOptions,
+                    filterText,
+                    items,
+                    multiple,
+                    value,
+                    itemId,
+                    groupBy,
+                    label,
+                    filterSelectedItems,
+                    itemFilter,
+                    convertStringItemsToObjects,
+                    filterGroupedItems,
+                })),
+        );
+    });
+    $effect.pre(() => {
+        prev_multiple = multiple;
+    });
+    $effect(() => {
+        hasValue;
+        untrack(() => {
+            if (items) setValue(items);
+        });
+    });
+    $effect(() => {
+        searchable;
+        untrack(() => {
+            if (inputAttributes || !searchable) assignInputAttributes();
+        });
+    });
+    $effect(() => {
+        untrack(() => {
+            if (multiple) setupMulti();
+        });
+    });
+    $effect(() => {
+        if (prev_multiple && !multiple) setupSingle();
+    });
+    $effect(() => {
+        if (multiple && value && value.length > 1) checkValueForDuplicates();
+    });
+    $effect(() => {
+        if (value) dispatchSelectedItem();
+    });
+    $effect(() => {
+        if (prev_value && !value) {
+            oninput(value || []);
+        }
+    });
+    $effect(() => {
+        if (!focused && input) closeList();
+    });
+    $effect(() => {
+        filterText;
+        untrack(() => {
+            if (filterText !== prev_filterText) setupFilterText();
+        });
+    });
+    $effect(() => {
+        if (!multiple && listOpen && value && filteredItems) setValueIndexAsHoverIndex();
+    });
+    $effect(() => {
+        dispatchHover(hoverItemIndex);
+    });
+    $effect(() => {
+        multiple;
+        value;
+        untrack(() => (hasValue = multiple ? value && value?.length > 0 : !!value));
+    });
+    $effect(() => {
+        hideSelectedItem = hasValue && filterText.length > 0;
+    });
+    $effect(() => {
+        showClear = hasValue && clearable && !disabled && !loading;
+    });
+    $effect(() => {
+        ariaSelection = value ? handleAriaSelection(multiple) : '';
+    });
+    $effect(() => {
+        ariaContext = handleAriaContent({ filteredItems, hoverItemIndex, focused, listOpen });
```

Comment:
`ariaContext` has no use.  This is either a mistake, or is really not needed.  If needed, it can be a `$derived`.